crates/core_simd/src/ops/unop.rs

```diff
@@ -0,0 +1,103 @@
+use crate::simd::intrinsics;
```

Comment:
Could we call this file unary.rs instead?

Reply:
Sure.